lib/reader.js

```diff
@@ -1,38 +1,9 @@
 'use strict';
 
-const stream = require('readable-stream');
-const isUndefined = require('lodash.isundefined');
 const JSONStream = require('JSONStream');
 const pack = require('browser-pack');
 const mergeStream = require('merge-stream');
-const depsSort = require('deps-sort');
-
-
-/**
- *
- * We need to manipulate order in all bundles to make sure that the
- * order is an increased value accross all bundles.
- *
- */
-
-class Order extends stream.Transform {
-    constructor () {
-        super({
-            objectMode: true,
-        });
-        this.order = 0;
-    }
-
-    _transform (obj, encoding, next) {
-        if (!isUndefined(obj.order)) {
-            obj.order = this.order;
-            this.order += 1;
-        }
-        this.push(obj);
-        next();
-    }
-}
-
+const through = require('through2');
```

Comment:
Since we use Classes and all the new stuff I would prefer to use native streams (readable-stream) instead of through. through is mostly there for making sub classing nicer pre Classes area.